frontend/templates/layouts/mail.html

```diff
@@ -0,0 +1,11 @@
+<html>
```

Comment:
can we rename the path to email/approved.html and have another one for email/rejected.html

we can add the rejected template which mentions the reason for rejection as well.